integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java

```diff
@@ -838,6 +866,162 @@ public void testErrorPathFilePermission() {
     deleteDomainResource(errorpathDomainNamespace, domainUid);
   }
 
+  /**
+   * Create a domain using multiple auxiliary images.
+   * One auxiliary image (image1) contains the domain configuration and
+   * another auxiliary image (image2) with WDT only,
+   * update WDT version by patching with another auxiliary image (image3)
+   * and verify the domain is running.
```

Comment:
Is there any way to verify that we are using the latest/patched WDT version ?

Reply:
added verification